projects/plugins/crm/modules/woo-sync/includes/class-woo-sync-background-sync-job.php

```diff
@@ -1705,6 +1725,39 @@ public function woocommerce_order_to_crm_objects(
 
 			}
 
+			// --- Process any present fee in the order --- //
+			$fees = $order->get_fees();
+
+			if ( is_array( $fees ) && count( $fees ) > 0 ) {
+				foreach ( $fees as $fee ) {
+					if ( $fee instanceof \WC_Order_Item_Fee ) {
+						$new_line_item = array(
+							'order'    => $order_post_id, // passed as parameter to this function
+							'currency' => $order_currency,
+							'quantity' => 1,
+							'price'    => $fee->get_amount( false ),
+							'fee'      => $fee->get_amount( false ),
+							'total'    => $fee->get_amount( false ),
+							'title'    => esc_html__( 'Fee', 'zero-bs-crm' ),
+							'desc'     => $fee->get_name(),
+							'tax'      => $fee->get_total_tax(),
+							'taxes'    => -1,
```

Comment:
Why default to `-1` (legacy bad practice) as opposed to not passing it at all?

Reply:
Yes, I don't like to use -1 either for storing a `null` status. But here it's acting as a simple code, it's not a proper value or an ID. It could have been `9999`.

Reply:
But again, here, consider `-1` as a status code to identify non-taxes. It is used by the `invoicebuilder.js` file to identify those cases via API. This value is not stored in the database. It could have been `0` or `false` too, or just empty. I had to adapt to this legacy code.

I'm not particularly eager to use it, but in this case, I'll leave it as it is.